charters/DataStore/charter.md

```diff
@@ -0,0 +1,72 @@
+
+# Data Store
+
+
+## Description
+The Data Store is a scientific data sharing/publishing/distribution framework, providing file/bundle management on multiple clouds at petabyte-scale. It defines public APIs for storage, retrieval, and subscription to events that functions transparently across multiple cloud systems such as AWS and GCP.
+
+## Objective
```

Comment:
Elsewhere in the document it is clear that the team intends to a) build software, and b) provide devops services in support of operating this software.    This section should make that explicit, as devops support (to the lead Ops team) is a core objective of the team.